website/sidebars.js

```diff
@@ -656,6 +656,7 @@ const sidebarSettings = {
             slug: '/guides/migration/versions',
           },
           items: [
```

Comment:
@jtcohen6 and @dbeatty10 what do you think about autogenerating this sidebar? Then we don't need to update sidebar.js every release. It would looks something [like this](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/sidebars.js#L493-L508)

Reply:
I don't have a preference. Whatever you both think is best.

Reply:
I like the idea of auto-generating! I don't think that needs to block this PR if it's work we can pick up separately.